app/assets/stylesheets/form.scss

```diff
@@ -0,0 +1,36 @@
+@import "foundation_and_overrides";
+$black: #000;
+$white: #ffffff;
+$background: #496D7B;
+
+.page-background {
+  background-color: $background;
+  margin-top: 1.5rem;
+}
+
+.form {
+  text-align: center;
```

Comment:
Properties should be ordered border, color, margin, margin-top, padding, text-align